spec/controllers/idv/hybrid_mobile/document_capture_controller_spec.rb

```diff
@@ -0,0 +1,250 @@
+          analytics_id: 'Doc Auth',
+          flow_path: 'hybrid',
+          irs_reproofing: false,
+          step: 'document_capture',
+        }
+      end
+
+      it 'renders the show template' do
+        expect(subject).to receive(:render).with(
+          :show,
+          locals: hash_including(
+            document_capture_session_uuid: document_capture_session_uuid,
+          ),
+        ).and_call_original
+
+        get :show
+
+        expect(response).to render_template :show
+      end
+
+      it 'sends analytics_visited event' do
+        get :show
+
+        expect(@analytics).to have_logged_event(analytics_name, analytics_args)
+      end
+
+      it 'updates DocAuthLog document_capture_view_count' do
+        doc_auth_log = DocAuthLog.create(user_id: user.id)
+
+        expect { get :show }.to(
+          change { doc_auth_log.reload.document_capture_view_count }.from(0).to(1),
+        )
+      end
+
+      context 'feature flag disabled' do
+        let(:feature_flag_enabled) { false }
+        it 'returns a 404' do
+          get :show
+          expect(response.status).to eql(404)
+        end
+      end
+
+      context 'with expired DocumentCaptureSession' do
+        let(:document_capture_session_requested_at) do
+          Time.zone.now.advance(
+            minutes: IdentityConfig.store.doc_capture_request_valid_for_minutes * -2,
+          )
+        end
+        it 'redirects to root' do
+          get :show
+          expect(response).to redirect_to(root_url)
+        end
+      end
+    end
+  end
+
+  describe '#update' do
+    before do
+      allow_any_instance_of(DocumentCaptureSession).to receive(:load_result).and_return(
+        DocumentCaptureSessionResult.new(
+          id: 1234,
+          success: true,
+          pii: {
+            state: 'WA',
+          },
+          attention_with_barcode: true,
+        ),
+      )
+    end
+
+    context 'with no user id in session' do
+      let(:user) { nil }
+      let(:document_capture_session) { nil }
+      it 'redirects to root' do
+        get :show
+        expect(response).to redirect_to root_url
+      end
+    end
+
+    context 'with a user id in session' do
+      before do
+        session[:doc_capture_user_id] = user.id
+      end
+
+      it 'tracks expected events' do
+        get :show
+        put :update
+
+        expect(@analytics).to have_logged_event(
+          'IdV: doc auth document_capture submitted',
+          hash_including(
+            flow_path: 'hybrid',
+            step: 'document_capture',
+            analytics_id: 'Doc Auth',
+          ),
+        )
+      end
+
+      it 'does not raise an exception when stored_result is nil' do
+        allow(FeatureManagement).to receive(:document_capture_async_uploads_enabled?).
+          and_return(false)
+
+        allow(subject).to receive(:stored_result).and_return(nil)
+
+        put :update
+      end
+
+      it 'redirects to complete step' do
```

Comment:
For a second I thought there was a new step I didn't know about!
```suggestion
      it 'redirects to CaptureComplete step' do
```